src/error.rs

```diff
@@ -1041,8 +1052,7 @@ impl IntoDiagnostics<FileId> for EvalError {
 
                 vec![Diagnostic::error()
                     .with_message(format!("{format} parse error: {msg}"))
-                    .with_labels(labels)
-                    .with_notes(vec![String::from(INTERNAL_ERROR_MSG)])]
+                    .with_labels(labels)]
```

Comment:
This can very much happen, for example `std.deserialize `Json "{]"`, and is not an internal error. I wonder if the INTERAL_ERROR_MSG weren't just a leftover from copy pasting code around.